modin/engines/dask/pandas_on_dask/frame/partition_manager.py

```diff
@@ -51,31 +75,31 @@ def deploy_func(df, apply_func, call_queue_df=None, call_queues_other=None, *oth
 
 
 class DaskFrameManager(BaseFrameManager):
-    """This class implements the interface in `BaseFrameManager`."""
+    """The class implements the interface in `BaseFrameManager`."""
 
-    # This object uses RayRemotePartition objects as the underlying store.
+    # This object uses PandasOnDaskFramePartition objects as the underlying store.
```

Comment:
do we even need this "thank you, captain Obvious" comment?..

Reply:
I think, yes, just because there is no need to go to the docs.